shared/reducers/wallets.tsx

```diff
@@ -26,618 +26,595 @@ const reduceAssetMap = (
-    case WalletsGen.rejectDisclaimer:
-    case WalletsGen.cancelPayment:
-    case WalletsGen.cancelRequest:
-    case WalletsGen.createNewAccount:
-    case WalletsGen.exportSecretKey:
-    case WalletsGen.linkExistingAccount:
-    case WalletsGen.loadAssets:
-    case WalletsGen.loadPaymentDetail:
-    case WalletsGen.loadPayments:
-    case WalletsGen.loadDisplayCurrencies:
-    case WalletsGen.markAsRead:
-    case WalletsGen.loadDisplayCurrency:
-    case WalletsGen.changeDisplayCurrency:
-    case WalletsGen.changeAccountName:
-    case WalletsGen.checkDisclaimer:
-    case WalletsGen.deleteAccount:
-    case WalletsGen.deletedAccount:
-    case WalletsGen.loadAccounts:
-    case WalletsGen.loadWalletDisclaimer:
-    case WalletsGen.setAccountAsDefault:
-    case WalletsGen.showTransaction:
-    case WalletsGen.sendPayment:
-    case WalletsGen.sentPayment:
-    case WalletsGen.requestPayment:
-    case WalletsGen.requestedPayment:
-    case WalletsGen.loadSendAssetChoices:
-    case WalletsGen.loadMobileOnlyMode:
-    case WalletsGen.changeMobileOnlyMode:
-    case WalletsGen.exitFailedPayment:
-    case WalletsGen.loadExternalPartners:
-    case WalletsGen.acceptSEP7Pay:
-    case WalletsGen.acceptSEP7Path:
-    case WalletsGen.acceptSEP7Tx:
-    case WalletsGen.refreshTrustlineAcceptedAssets:
-    case WalletsGen.refreshTrustlinePopularAssets:
-    case WalletsGen.calculateBuildingAdvanced:
-      return state
-    default:
-      return state
-  }
-}
+  ),
+})
 
 if (__DEV__) {
-    console.log(new Array(100).fill('wallets reducer double check').join('\n'))
+  console.log(new Array(100).fill('wallets reducer double check').join('\n'))
 }
 
 const doubleCheck = (
   state: Types.State = initialState,
   action: WalletsGen.Actions | TeamBuildingGen.Actions
 ): Types.State => {
-    const nextState = newReducer(state, action)
+  const nextState = newReducer(state, action)
 
-    if (__DEV__) {
-        // TODO transform
-        const s = ConstantsOLD.makeState(state)
-        const nextStateOLD   = reducerOLD(s,action)
-        // TODO compare
+  if (__DEV__) {
+    const s = ConstantsOLD.makeState(state)
+    const nextStateOLD = reducerOLD(s, action)
+    const o: any = {
+      ...nextStateOLD.toJS(),
+      reviewLastSeqno: nextStateOLD.reviewLastSeqno || null,
+      sep7ConfirmInfo: nextStateOLD.sep7ConfirmInfo || null,
+      staticConfig: nextStateOLD.staticConfig || null,
     }
 
-    return nextState
+    const n: any = {
+      ...nextState,
+      reviewLastSeqno: nextState.reviewLastSeqno || null,
+      sep7ConfirmInfo: nextState.sep7ConfirmInfo || null,
+      staticConfig: nextState.staticConfig || null,
+    }
+
+    let same = true
+    Object.keys(o).forEach(k => {
+      const so = JSON.stringify(o[k])
+      const sn = JSON.stringify(n[k])
+      if (so !== sn) {
+        same = false
+        console.log('aaa diff', k, so, sn, action)
+      }
+    })
+    if (same) {
+      console.log('aaa same')
```

Comment:
temp logging? I haven't looked at the rest of these so maybe it's already removed.